plugins/woocommerce/src/Internal/Admin/Settings.php

```diff
@@ -212,7 +212,10 @@ public function add_component_settings( $settings ) {
 		$settings['woocommerceTranslation'] = __( 'WooCommerce', 'woocommerce' );
 		// We may have synced orders with a now-unregistered status.
 		// E.g An extension that added statuses is now inactive or removed.
-		$settings['unregisteredOrderStatuses'] = $this->get_unregistered_order_statuses();
+		 if(PageController::is_admin_page()){
```

Comment:
There's a mixture of tabs and spaces on this line.  Can you update this to only use tabs?  Also the linter is complaining about lack of spaces around the if statement parentheses.

```suggestion
		if( PageController::is_admin_page() ){
```